Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
On Mon, May 23, 2016 at 12:57:53PM -0700, Vincent Batts wrote:
> > @@ -1,6 +1,5 @@
> >  
> >  DOCKER ?= $(shell command -v docker)
> > -PANDOC ?= $(shell command -v pandoc)
> 
> and also making the path to the input and output files

Ah, yes.  So easiest way to support local pandoc is probably leave
PANDOC variable here and make the docs rules conditional.  If PANDOC
is non-empty, use:

  $(PANDOC) -f markdown_github -t latex -o $@ $(DOC_FILES)

and if PANDOC is empty use the current $(DOCKER) invocation with the
hard-coded vbatts/pandoc image.

The alternative is to make variables for the input and output
directories and set them conditionally depending on whether PANDOC is
empty.